ajax/libs/raty/package.json

```
@@ -23,17 +23,14 @@
     "url": "https://github.com/wbotelhos/raty"
   },
   "version": "2.8.0",
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/wbotelhos/raty.git",
-    "fileMap": [
-      {
-        "basePath": "lib",
-        "files": [
-          "**/?(jquery.)raty*",
-          "images/*.png"
-        ]
-      }
-    ]
-  }
+  "npmName": "raty-js",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/?(jquery.)raty*", 
+        "images/*.png"
```

```suggestion
        "**/*", 
```
The fonts might also be needed, so a glob for everything in `lib` seems best.